widgets/common/src/filters/inputs/MultipleStringValuesInput.tsx

```diff
@@ -0,0 +1,35 @@
+import { FunctionComponent, useMemo } from 'react';
+
+interface MultipleStringValuesInputProps {
+    name: string;
+    onChange: (newValue: string[]) => void;
+    placeholder: string;
+    value: string[];
```

Comment:
Optional:

I prefer using plurals for arrays :smile: 

```suggestion
    values: string[];
```

Reply:
Hmm... If we would do it so here, then I think we should do it as well in the other places:
![image](https://user-images.githubusercontent.com/5202105/114835892-57671b00-9dd2-11eb-9704-fcc7c58fc468.png)

As this will be a bit bigger change, I would leave it for now and maybe consider changing it later.

Reply:
I see :smile: We can ignore the matter here, but generally, does it matter that we always make changes in a single step? I don't see it as problematic that we would have it a prop named `values` and we would pass `data.value` to it :slightly_smiling_face: I find it easier to make small improvements when it's possible, even if that means that we break some existing pattern (that we can agree can be improved)

Reply:
I'm not sure if I understand the message entirely. In general more often I'm leaning towards having consistent application, instead of perfectly optimized parts. Maybe that is something to discuss in the weekly 🙂 Maybe that's just the preferences difference which we can live with 🙂

Reply:
Sure, I've added a point about it :slightly_smiling_face: